conf.d/pure.fish

```diff
@@ -33,6 +33,9 @@ _pure_set_default pure_color_git_unpushed_commits pure_color_info
 _pure_set_default pure_color_git_branch pure_color_mute
 _pure_set_default pure_color_git_dirty pure_color_mute
 _pure_set_default pure_color_git_stash pure_color_info
+_pure_set_default pure_show_numbered_git_unpulled_commits false
+_pure_set_default pure_show_numbered_git_unpushed_commits false
+_pure_set_default pure_show_numbered_git_stash false
```

Comment:
Asking for opinion: Not sure if these config is too granular. Would it be better if instead we only have one config `pure_show_numbered_git_indicator`?

Reply:
Thanks for the PR, having a single flag seems more reasonable indeed. I'm not really fond of the idea of so much customization, as it will add complexity.

So let's simplify to a single `pure_show_numbered_git_indicator` flag

Reply:
Thanks for the input! Updated to have one merged flag.